rust/datafusion/src/execution/physical_plan/csv.rs

```diff
@@ -163,6 +163,8 @@ impl Partition for CsvPartition {
 
 /// Iterator over batches
 struct CsvIterator {
+    /// Schema
+    schema: Arc<Schema>,
```

Comment:
The reader already exposes a schema (https://github.com/apache/arrow/blob/master/rust/arrow/src/csv/reader.rs#L221), so wouldn't this be redundant as we could get the schema directly from `reader`?

Reply:
Good catch. Fixed.